url_filter/filtersets/django.py

```diff
@@ -122,8 +122,14 @@ def _build_filterset_from_related_field(self, field):
         Build a :class:`.FilterSet` for a Django relation model field
         such as ``ForeignKey``.
         """
+        try:
+            name = field.remote_field.name
```

Comment:
usually this results is lower coverage when running against a particular version of a dep. maybe do something like this?

```python
name = (getattr(field, 'remote_field', None) or field.rel).name
```

the `or` to make the evaluation lazy in case first one succeeds

I know this is not exactly pythonic but I guess Im a sucker for the coveted 100% coverage locally...

Reply:
Sure.